internal/metrics/buckets.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+// NonNegativeBuckets provides rounded bucket boundaries for histograms
+// that will only store non-negative values.
+//
+// Note that these buckets are intended to be used only by the metrics generated
+// by the weaver runtime.
+var NonNegativeBuckets = []float64{
```

Comment:
Maybe move to `generated.go` and rename to `GeneratedBuckets`, so it's clear?

Reply:
Sounds good. Done